policyengine_us/variables/gov/states/mt/tax/income/base/mt_agi.py

```diff
@@ -16,10 +16,21 @@ def formula(person, period, parameters):
         reduced_agi = max_(agi + additions - subtractions, 0)
         # Montana taxable social security benefits can be either addition or subtraction
         # if mt_taxable_social security - taxable_social_security > 0, then addition, else subtraction
-        taxable_ss = person("taxable_social_security", period)
-        mt_taxable_ss = person("mt_taxable_social_security", period)
-        adjusted_mt_ss_difference = mt_taxable_ss - taxable_ss
-        tax_unit_mt_agi = max_(reduced_agi + adjusted_mt_ss_difference, 0)
+
+        # Get the current year
+        year = period.start.year
+
+        if year <= 2023:
```

Comment:
lets parameterize the if statements so we can show the impacts of the change by switching it on/off in the web app 

Similar logic:
```
if p.apples: 
...
```

Reply:
similar below